[BUG] Cannot find name '$localize' when running Storybook

**Describe the bug**
_Storybook_ is not starting anymore.

**To Reproduce**
Steps to reproduce the behavior:
1. Run `yarn start:storybook`
2. Get various errors in files of `libs/ui/src` like: `Cannot find name '$localize'.`

**Expected behavior**
_Storybook_ starts without errors and can be accessed via http://localhost:4400/

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

**Logs**
```
Error: libs/ui/src/lib/activities-table/activities-table.component.ts:134:7 - error TS2304: Cannot find name '$localize'.
$localize`Do you really want to delete this activity?`
at addError (/dev/ghostfolio/source/node_modules/@angular-devkit/build-angular/node_modules/@ngtools/webpack/src/ivy/diagnostics.js:31:29)
```
**Environment**
 - Ghostfolio Version `1.201.0`

--- a/libs/ui/.storybook/preview.js
+++ b/libs/ui/.storybook/preview.js
@@ -1,3 +1,1 @@
-import '@angular/localize/init';
-
 import '!style-loader!css-loader!sass-loader!../../../apps/client/src/styles.scss';

--- a/libs/ui/src/lib/fire-calculator/fire-calculator.component.stories.ts
+++ b/libs/ui/src/lib/fire-calculator/fire-calculator.component.stories.ts
@@ -1,3 +1,4 @@
+import '@angular/localize/init';
 import { CommonModule } from '@angular/common';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatButtonModule } from '@angular/material/button';

